debug/animation.html

```diff
@@ -77,21 +77,21 @@
 
     const s = carto.expressions;
     const duration = 6;
-    const torque = s.torque(
+    const animation = s.animation(
       s.prop('sim_time'),
       duration,
       s.fade(duration/10, duration/5)
     );
     const viz = new carto.Viz({
         strokeWidth: 1,
-        color: s.hsv(torque, 1, 1),
-        width: s.add(1,s.mul(torque,30)),
+        color: s.hsv(animation, 1, 1),
+        width: s.add(1,s.mul(animation,30)),
         order: s.desc(s.width())
     });
     const layer = new carto.Layer('layer', source, viz);
     layer.addTo(map, 'background');
     layer.on('updated', () => {
-        const time = torque.getSimTime().toISOString();
+        const time = animation.getCurrenValue().toISOString();
```

Comment:
I wouldn't name it `getCurrentValue` :confused: Anyway, there is a missing `t`

Reply:
True. It is difficult to find a name, how would we name it?

* ~~getSimTime~~
* ~~getTimestamp~~
* ~~getCurrentValue~~
* getCurrentTime
* getValue
* getTime
* ?

Reply:
`getPosition`?

Reply:
IMO is not *actually* the position... what about `getProgressValue`?

Reply:
`getProgressValue` for this and `getProgressPct` for the [0,1] one?